### PR TITLE
update pin

### PR DESCRIPTION
**Pull request by ocefpaf**

### CODE REVIEW

**Comment by ocefpaf** [comment]
`OS X` is not linking to `openblas` :confused: 

```
+ conda inspect linkages -p /Users/travis/miniconda3/conda-bld/numpy_1501077752784/_t_env numpy
numpy
-----
system:
    Accelerate (/System/Library/Frameworks/Accelerate.framework/Versions/A/Accelerate)
    libSystem.B.dylib (/usr/lib/libSystem.B.dylib)
    libc++.1.dylib (/usr/lib/libc++.1.dylib)
not found:
```

I'll investigate this after lunch.

**Comment by jakirkham** [comment]
That seems to the be the case with the current [build]( https://travis-ci.org/conda-forge/numpy-feedstock/builds/255390785#L3994 ) of NumPy 1.7 as well.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,11 +24,11 @@ requirements:
     - python
     - setuptools
     - blas 1.1 {{ variant }}
-    - openblas 0.2.19|0.2.19.*
+    - openblas 0.2.20  # [linux]
   run:
     - python
     - blas 1.1 {{ variant }}
-    - openblas 0.2.19|0.2.19.*
+    - openblas >=0.2.15  # [linux]
```
Comment (isuruf) [inline recipe/meta.yaml]: Why only linux?

Comment (isuruf) [inline recipe/meta.yaml]: Ah, it's not linking

Comment (ocefpaf) [inline recipe/meta.yaml]: Yep. I'll investigate that later but it won't affect us. Virtually everything that links to `openblas`, like `scipy`,  needs `numpy >1.7` anyways.

**Pull request merged**